packages/inventory/package.json

```diff
@@ -41,11 +42,15 @@
   "private": false,
   "dependencies": {
     "@opensrp/location-management": "^0.0.7",
+    "@opensrp/pkg-config": "^0.0.1",
     "@opensrp/react-utils": "^0.0.2",
-    "axios": "^0.21.1"
+    "i18next": "^19.8.4"
   },
   "devDependencies": {
     "@onaio/session-reducer": "^0.0.12",
+    "i18next-parser": "^3.5.0",
+    "@opensrp/react-utils": "^0.0.2",
+    "axios": "^0.21.1",
```

Comment:
should this 2 be in devDependencies

Reply:
@p-netm I have removed react-utils  but axios is use in some tests in inventory

Reply:
I think both are used but as runtime dependencies

Reply:
so they should be in dependencies